.github/workflows/esti.yaml

```diff
@@ -757,23 +757,36 @@ jobs:
         working-directory: test/spark
         run: docker-compose logs --tail=1000 lakefs
 
-      - name: Test lakeFS S3 with Spark 3.x thick client
+      - name: Spark${{ matrix.spark.tag }} + lakeFS HadoopFS
         timeout-minutes: 8
         working-directory: test/spark
         run: |
           python ./run-test.py \
             --storage_namespace s3://esti-system-testing/${{ github.run_number }}-spark${{ matrix.spark.tag }}-client/${{ steps.unique.outputs.value }} \
             --repository thick-client-test \
             --sonnet_jar ${{ matrix.spark.sonnet_jar }} \
-            --direct_access \
+            --access_mode hadoopfs \
             --aws_access_key ${{ secrets.ESTI_AWS_ACCESS_KEY_ID }} \
             --aws_secret_key ${{ secrets.ESTI_AWS_SECRET_ACCESS_KEY }}
-      - name: lakeFS Logs on Spark with client failure
+      - name: lakeFS Logs on HadoopFS test failure
+        if: ${{ failure() }}
+        continue-on-error: true
+        working-directory: test/spark
+        run: docker-compose logs --tail=1000 lakefs
```

Comment:
Please can we have _many_ more log lines?

Reply:
Opened #5537.